Docker_files/ocsci_container/scripts/install-ocs-ci.sh

```diff
@@ -0,0 +1,34 @@
+#!/bin/bash
+
+set -x
+
+OCSCI_INSTALL_DIR="${OCSCI_INSTALL_DIR:=/opt/ocs-ci}"
+OCSCI_REPO_URL="https://github.com/red-hat-storage/ocs-ci"
+
+# Clone OCS-CI Project
+git clone "$OCSCI_REPO_URL" "$OCSCI_INSTALL_DIR"
+pushd "$OCSCI_INSTALL_DIR"
+
+# Checkout to relevant branch [the default is stable]
+if [ "$BRANCH_ID" == "" ]
+then
+      echo checkout to stable branch
+      git checkout stable
+elif [[ $BRANCH_ID == "master" ]]
+then
+      echo Working on master branch
+else
+      echo Checkout to PR $BRANCH_ID
+      git fetch origin pull/$BRANCH_ID/head:$BRANCH_ID
+      git checkout $BRANCH_ID
+fi
+
+#Install virtual environment
+python3.8 -m venv venv
```

Comment:
I think that we discussed this offline as well that we can avoid creating venv inside container

Reply:
It is not working
```
$ docker images 
REPOSITORY                    TAG       IMAGE ID       CREATED          SIZE
oded_image1                   latest    249d168b922b   20 minutes ago   1.58GB
registry.redhat.io/ubi8/ubi   latest    16fb1f57c5af   4 weeks ago      207MB

 [odedviner@fedora ocsci_container]$ docker run -it oded_image1 /bin/bash
[root@4c18853ba141 ocs-ci]# pip install --upgrade pip setuptools
bash: pip: command not found
```

Reply:
I think you need to use pip3.8

Reply:
correct:)
done.